### PR TITLE
Created readme and first two sections and styling

### PR DESCRIPTION
**Pull request by AlyAtalla**
Created readme file and Added two sections to the HTML file with appropriate styling using flexbox.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <header>
+    <nav class="nav">
+        <div class="menu-button">
+            <i class="fa-solid fa-bars fa-xl"></i>
+        </div>
+        <div class="header-links">
+            <a class="header-links" href="#works">Experience</a>
+            <a class="header-links" href="#about">About</a>
+            <a class="header-links" href="#contact-me">Contact</a>
+        </div>
```
Comment (souad988) [inline index.html]: kindly wrap this list of links in ul li tags this is important for semantic reasons

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @AlyAtalla , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Professional README 
No linter errors

## Required Changes ♻️
_Check the comments under the review._
_Also please fix some small styling issues to make your project llok exactly like the design 

1. the background is not correctly aligned
2. font size of the headlines p tag need to be increased
3. and the margins between headlines needs be decreased


image (3)
image (4)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlyAtalla** [comment]
Sorry for closing the pull request, I am still getting familiar with the workflow. thank you for your effort.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  <header>
+    <nav class="nav">
+        <div class="menu-button">
+            <i class="fa-solid fa-bars fa-xl"></i>
+        </div>
+        <ul class="header-links">
+            <li class="header-links" href="#works">Experience</li>
+            <li class="header-links" href="#about">About</li>
+            <li class="header-links" href="#contact">Contact</li>
+        </ul>
```
Comment (wayungi) [inline index.html:21]: - [ ] Good work implementing the required changes here. You did not need to delete the anchor tags. Kindly make the changes like below.
```
<ul>
   <li><a class="header-links" href="#works">Experience</a></li>
   <li><a class="header-links" href="#about">About</a></li>
   <li><a class="header-links" href="#contact-me">Contact</a></li>
</ul>
```

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @AlyAtalla , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
 - [ ] Linters correctly set up 👍🏾 
 - [ ] Clean code, no comments 👍🏾 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Some changes as requested by the previous reviewer have not been implemented. Please see below the areas circled and fix them to match the Figma

**Figma**
figma

**Your design**
figma2

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Alusp) [reply index.html:21]: - [ ][Required] Good job on implementing the changes so far, However, As requested by the previous Code Reviewer Please add an anchor Tag in `<li>`, you can use this [link](https://www.w3schools.com/tags/tag_a.asp) to know how to implement it correctly. or you can follow exactly as it is in his code example :smile:

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @AlyAtalla , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost There


 ## Highlights

- [ ] No Linter Errors :white_check_mark: 
- [ ] Good commit Message :white_check_mark: 
- [ ] Professional Readme used :white_check_mark: 


 ## Required Changes ♻️
_Check the comments under the review._

- [ ][Required] Please follow the Design Template as requested by the previous reviewer, Design templates provide a consistent structure and visual language across different elements of a project. This consistency helps create a cohesive and unified experience for the audience. By using consistent fonts, colors, layouts, and other design elements, you can establish a recognizable brand identity and improve overall user experience. :smile: 

You can see the difference in the table below showing your design and the Template. :arrow_down: 
<table>
    <tr>
      <td>Your Design</td>
      <td>The Template</td>
    </tr>
    <tr>
      <td>

Student Design
</td>
      <td>

Template Review

</td>
    </tr>
  </table>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @AlyAtalla , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Grat work on implementing all the required changes as requested by the previous reviewers :+1: 
- [ ] Great and good looking UI :art: :1st_place_medal: 
- [ ] All linters checks are passing :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**